src/Features/Core/Portable/AddImport/SymbolReferenceFinder.cs

```diff
@@ -463,6 +464,28 @@ private async Task<ImmutableArray<SymbolReference>> GetReferencesForQueryPattern
                 return ImmutableArray<SymbolReference>.Empty;
             }
 
+            /// <summary>
+            /// Searches for extension methods exactly called 'GetAwaiter'.  Returns
+            /// <see cref="SymbolReference"/>s to the <see cref="INamespaceSymbol"/>s that contain
+            /// the static classes that those extension methods are contained in.
+            /// </summary>
+            private async Task<ImmutableArray<SymbolReference>> GetReferencesForGetAwaiterAsync(SearchScope searchScope)
+            {
+                searchScope.CancellationToken.ThrowIfCancellationRequested();
+
+                if (_owner.CanAddImportForGetAwaiter(_diagnosticId, _node))
+                {
+                    var type = _owner.GetAwaitInfo(_semanticModel, _node, searchScope.CancellationToken);
+                    if (type != null)
+                    {
+                        return await GetReferencesForExtensionMethodAsync(searchScope, "GetAwaiter", type,
+                            m => m.IsValidGetAwaiter()).ConfigureAwait(false); 
```

Comment:
any existing GetAwaiter you could use nameof on instead of hardcoding this string?

Reply:
Good point, I should've looked harder. Looks like there's a perfectly useful `WellKnownMemberNames` class that I've been neglecting